.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -10,7 +10,7 @@ CADET-Core release
 CADET-Core releases follow the *semantic versioning system*, which is documented [here](https://semver.org/).
 
 **CADET-Core is released following the so-called *GitLab flow*:**
-In GitLab flow, feature and hotfix branches contain work for new features and bug fixes which will be merged back into the master branch immediately when they’re finished, reviewed, and approved. The master branch is ready to be deployed, but not necessarily the source of truth for a new release: Each release has an associated release branch that is based off the master branch. Changes that are intended to be released are cherry-picked from master into the release branch. The release is made on the release branch. If desired, some release branches can be kept and maintained, which is currently not planned for the CADET-Core project.
+In GitLab flow, feature and hotfix branches contain work for new features and bug fixes which will be merged back into the master branch immediately when they’re finished, reviewed, and approved. The master branch is ready to be deployed, but not necessarily the source of truth for a new release: Each release has an associated release branch that is based off the master branch. Changes that are intended to be released are cherry-picked from master onto the release branch. The release is made on the release branch. Some release branches can be kept and maintained, e.g. branch `v5.0.X` to support the last major release.
```

Comment:
> last major release

Wouldn't `v5.0.X` be there to maintain the latest minor release?

Side note: using individual lines for each sentence would make reviews a bit more conveninent

Reply:
`v5.2.X` would support a minor release; `v5.0.X` supports the major release, right?
I know, usually I would make a single line for each sentence, but there was a reason against linebreaks, maybe the import from the rst file.